app/models/pipeline_run_stage.rb

```diff
@@ -233,13 +233,16 @@ def host_filtering_command
     # Upload DAG to S3
     sample = pipeline_run.sample
     file_ext = sample.fasta_input? ? 'fasta' : 'fastq'
+    nucleotide_type = sample.metadata && sample.metadata[:nucleotide_type] ? sample.metadata[:nucleotide_type] : '',
```

Comment:
I am pretty confident this is incorrect. Can I get some feedback on this?

Reply:
Wrote and tested this for you :)
```
nucleotide_type_metadatum = sample.metadata.find_by(key: "nucleotide_type")
nucleotide_type = nucleotide_type_metadatum ? nucleotide_type_metadatum.string_validated_value : ''
```